cri-purge.sh

```diff
@@ -21,9 +21,11 @@ VERSION="0.07"
 
 ###[ Define Variables ]#######################################################
 CRI_CMD="crictl"
+CRIINFO_CMD="crio-status"
 
-# Skip images with these tags, let human deal with them
-SKIP_THESE_TAGS="<none> latest"
+# Skip images with these tags, let human deal with them, for example:
+# SKIP_THESE_TAGS="<none> latest"
+SKIP_THESE_TAGS="^\S+\s+[\w-_\.\d]+\s+"
```

Comment:
Can you describe which tags this change would now skip? When I use the "-s" option to list images being skipped I get an error "grep: invalid range in character class" with this new value.

Reply:
I figured out how the regex works, very nice! However, to fix the message "grep: invalid range in character class" I had to escape the "-" after the "w" to be like "\w\-_".